03.expression_tree/ExpressionTrees.Task1.ExpressionsTransformator/ChangeParameterVisitor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Linq.Expressions;
+
+namespace ExpressionTrees.Task1.ExpressionsTransformer
+{
+    public class ChangeParameterVisitor : ExpressionVisitor
+    {
+        private Dictionary<string, object> _parameters;
+
+        public Expression<T> ChangeParameter<T>(Expression<T> expression, Dictionary<string, object> parameters)
+        {
+            if(parameters != null)
```

Comment:
Null check is good, but what do you do if `parameters` is null? Not setting anything to `_parameters` will not prevent any errors, as it will remain equal null. Try it, you will get an exception.